indico/modules/designer/forms.py

```diff
@@ -16,15 +16,25 @@
 
 from __future__ import unicode_literals
 
-from wtforms import StringField
+from wtforms import StringField, BooleanField
 from wtforms.validators import DataRequired
 
 from indico.modules.designer.models.templates import TemplateType
+from indico.modules.events.models.events import Event
 from indico.util.i18n import _
 from indico.web.forms.base import IndicoForm
 from indico.web.forms.fields import IndicoEnumSelectField
+from indico.web.forms.widgets import SwitchWidget
 
 
 class AddTemplateForm(IndicoForm):
     title = StringField(_('Title'), [DataRequired()])
     type = IndicoEnumSelectField(_('Template'), enum=TemplateType, default=TemplateType.poster)
+    is_clonable = BooleanField(_('Allow cloning'), widget=SwitchWidget(), default=True,
+                               description=_("Allow event managers to clone this template."))
+
+    def __init__(self,  *args, **kwargs):
+        target = kwargs.pop('target')
+        super(AddTemplateForm, self).__init__(*args, **kwargs)
+        if isinstance(target, Event):
+            del self.is_clonable
```

Comment:
I'd do that in the RH after instantiating the form. You could even have an `always_clonable` attribute on `RHAddEventTemplate` / `RHAddCategoryTemplate` and use that flag to determine whether to remove the field from the form and whether to use the field's value when actually creating the template.